### PR TITLE
eman2: new package at 2.3

### PR DESCRIPTION
**Pull request by codeandkey**
The EMAN2 package is very dependent on anaconda/miniconda (it is included in the cmake build process), and it seems very difficult to decouple it from the build.

This package uses the conda-based installer, which installs a bunch of dependencies alongside the package. Not sure how well these types of packages fit in Spack -- feedback is appreciated!

### CODE REVIEW

**Comment by baberlevi** [comment]
Seems to be a source release here that will build with cmake: https://github.com/cryoem/eman2/releases. Might be worth trying out.

**Comment by baberlevi** [comment]
would also like to have a +cuda variant

**Comment by codeandkey** [comment]
The CMakeLists in the source repo checks for a miniconda installation (not sure why) -- The binary distribution should be CUDA-enabled when valid GPUs are present

**Comment by baberlevi** [comment]
@codeandkey is working on a version built from the source repo

File: var/spack/repos/builtin/packages/eman2/package.py [hunk 1]
```diff
@@ -6,39 +6,70 @@
...
-class Eman2(Package):
-    """EMAN2 is the successor to EMAN1. It is a broadly based greyscale
-       scientific image processing suite with a primary focus on processing
-       data from transmission electron microscopes."""
+class Eman2(CMakePackage):
+    """FIXME: Put a proper description of your package here."""
 
-    homepage = "https://blake.bcm.edu/emanwiki/EMAN2"
-    url      = "https://cryoem.bcm.edu/cryoem/static/software/release-2.3/eman2.3.linux64.sh"
+    # FIXME: Add a proper url for your package's homepage here.
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/eman2/package.py]: couple 'FIXME's to fix

Comment (codeandkey) [reply var/spack/repos/builtin/packages/eman2/package.py]: Oops, thanks -- fixed

**Comment by codeandkey** [comment]
As of right now this package will build and install but crashes at runtime due to an MPI linking issue:
`libEM2.so: undefined symbol: _ZN3MPI8Datatype4FreeEv`

**Comment by adamjstewart** [comment]
@codeandkey is this PR still a work in progress or is it abandoned?

**Comment by codeandkey** [comment]
Did not end up getting this to work in spack. Closing.

**Pull request closed without merging**